config/sequelize.js

```diff
@@ -1,6 +1,7 @@
 exports.default = {
     sequelize: function(api){
         return {
+            "autoMigrate" : true,
```

Comment:
add `"autoMigrate": false` in the commented out production example below, and I'll be sold!